emop-core/src/main/java/edu/cornell/emop/util/Util.java

```diff
@@ -102,4 +129,24 @@ public static Set<String> retrieveProjectPackageNames(File classesDir) {
         Set<String> fullSet = classFilesWalk(classesDir, classesDir.getAbsolutePath());
         return fullSet;
     }
+
+    public static void generateNewMonitorFile(String monitorFilePath, Set<String> specsToMonitor)
+            throws MojoExecutionException {
+        try (PrintWriter writer = new PrintWriter(monitorFilePath)) {
+            // Write header
+            writer.println("<aspectj>");
+            writer.println("<aspects>");
+            // Write body
+            for (String affectedSpec : specsToMonitor) {
```

Comment:
Could we call this something besides affectedSpec? So we don't assume RPS or anything